opm/output/eclipse/AggregateConnectionData.cpp

```diff
@@ -97,6 +103,26 @@ namespace {
         }
     }
 
+    template <class ConnOp>
+    void wellConnectionLoop(const Opm::Schedule&    sched,
+                            const std::size_t       sim_step,
+                            const Opm::EclipseGrid& grid,
+                            const Opm::data::Wells& xw,
+                            ConnOp&&                connOp,
+                            const std::string&      lgr_tag)
```

Comment:
I would prefer if the call-back function (`connOp`) was the last argument to this function.  That's more in line with other functions, e.g., from the `<algorithm>` header, that take these kinds of call-backs.